src/main/java/org/jenkinsci/plugins/ewm/facets/DiskStatsInfoFacet.java

```diff
@@ -0,0 +1,27 @@
+package org.jenkinsci.plugins.ewm.facets;
+
+import hudson.model.Fingerprint;
+import jenkins.model.FingerprintFacet;
+import org.jenkinsci.plugins.ewm.model.ExternalWorkspace;
+
+import javax.annotation.Nonnull;
+
+/**
+ * TODO JAVADOC
+ *
+ * @author Alexandru Somai
+ */
+public class DiskStatsInfoFacet extends FingerprintFacet {
```

Comment:
@oleg-nenashev do you think that I still need both `FingerprintFacet`s? Maybe the `WorkspaceBrowserFacet` should be enough? and I can serve the `ExternalWorkspace` object from there?